zebra-utils/zcash-rpc-diff

```diff
@@ -192,6 +195,37 @@ if [ "$1" == "getaddressbalance" ]; then
     fi
 fi
 
+if [ "$1" == "getrawmempool" ] && [ $CHECK_EXIT_STATUS != 0 ]; then
+    echo
+    echo "Obtaining transaction details"
+    echo
+
+    set TRANSACTION_ID
+    set TRANSACTION_HEX
+    set TRANSACTION_DECODED
+
+    while IFS= read -r line
+    do
+        if [ "$line" != "[" ] && [ "$line" != "]" ]; then
+            if [[ "${line:0-1}" == "," ]]; then
+                TRANSACTION_ID="${line:3:-2}"
+            else
+                TRANSACTION_ID="${line:3:-1}"
+            fi
+
+            TRANSACTION_HEX=`$ZCASH_CLI -rpcport="$ZEBRAD_RPC_PORT" getrawtransaction $TRANSACTION_ID 0`
```

Comment:
In some cases we might need the extra transactions from both nodes, and then the common transactions from just one of them.

So it would be something like:
- if Zebra is missing a transaction, get it from zcashd
- if zcashd is missing a transaction, get it from Zebra
- for all the other transactions, it doesn't matter which node

For example, if there are conflicting transactions on the network, Zebra might choose one, and zcashd might choose the other, and that difference is ok. But we won't be able to tell unless we have both transactions.

Reply:
I just dumped all transactions from both zcashd and zebrad in https://github.com/ZcashFoundation/zebra/pull/6035/commits/04e5b5722012fe0eaf7fe74370a1bb96262cdf1b

We could avoid the repeated ones in the second loop if needed.

Reply:
I think that's fine for now, we can do that later if we need to.